luarules/gadgets/unit_stockpile_limit.lua

```diff
@@ -28,14 +28,19 @@ if gadgetHandler:IsSyncedCode() then
 	local GiveOrderToUnit	= Spring.GiveOrderToUnit
 
 	local canStockpile = {}
+	Spring.Log(gadget:GetInfo().name, LOG.WARNING, "Check all stockpiles "..#UnitDefs)
```

Comment:
This is not a warning, this should be `LOG.INFO`, but should really just be removed, it adds noise to the log.

Reply:
Removed